### PR TITLE
[BPK-231] Add docs for srcset use in Image component

### PR DESCRIPTION
**Pull request by georgegillams**
+ [x] For any new components I've made sure that the style can be extended by the consumer using a `className` override.
+ [x] For any new files I've included the [Apache 2.0 License header](https://github.com/Skyscanner/backpack/blob/master/packages/bpk-tokens/formatters/license-header.js#L2-L16) at the top of the file.
+ [x] I've updated `changelog.md` for any changes that need to be highlighted in the changelog.
+ [x] If I've updated `npm-shrinkwrap.json` those changes are intentional.

### CODE REVIEW

File: packages/bpk-component-image/src/__snapshots__/BpkImage-test.js.snap [hunk 1]
```diff
@@ -545,3 +545,107 @@ exports[`BpkImage should render correctly 1`] = `
   </div>
 </div>
 `;
+
+exports[`BpkImage should support srcSet 1`] = `
```
Comment (matthewdavidson) [inline packages/bpk-component-image/src/__snapshots__/BpkImage-test.js.snap]: this snapshot doesn't assert against any sort of `srcset` support

File: packages/bpk-component-image/stories.js [hunk 2]
```diff
@@ -39,14 +39,35 @@ storiesOf('bpk-component-image', module)
...
+      height={408}
+      src={image}
+    />
+  ))
+  .add('Using SrcSet', () => (
+    <BpkImage
+      altText="image"
+      width={612}
+      height={408}
+      srcSet={`${image}?resize=280px:280px&quality=100 280w,
```
Comment (matthewdavidson) [inline packages/bpk-component-image/stories.js]: AFAIK you still need to supply a `src` attribute when using `srcset`...

File: packages/bpk-docs/src/pages/ImagesPage/ImagesPage.js [hunk 3]
```diff
@@ -48,7 +50,24 @@ const components = [
         altText="Lake"
         width={816}
         height={544}
-        src={`/${image1}`}
+        src={image1}
+        srcSet={`${image1}?resize=280px:280px&quality=100 280w,
+          ${image1}?resize=320px:320px&quality=100 320w,
```
Comment (matthewdavidson) [inline packages/bpk-docs/src/pages/ImagesPage/ImagesPage.js]: All these examples contain quite a lot of different options - two or three should be adequate, no?

File: packages/bpk-component-image/stories.js (hunk 2 above)
Comment (georgegillams) [reply packages/bpk-component-image/stories.js]: FYI it works without, but probably best practice to!

Comment (matthewdavidson) [reply packages/bpk-component-image/stories.js]: Yup, as a fallback for older browsers 👍

File: packages/bpk-component-image/src/__snapshots__/BpkImage-test.js.snap (hunk 1 above)
Comment (georgegillams) [reply packages/bpk-component-image/src/__snapshots__/BpkImage-test.js.snap]: Ah - `inView` was set to false!

File: packages/bpk-docs/src/pages/ImagesPage/ImagesPage.js (hunk 3 above)
Comment (georgegillams) [reply packages/bpk-docs/src/pages/ImagesPage/ImagesPage.js]: Fair point - have reduced to 4

File: packages/bpk-docs/src/pages/ImagesPage/ImagesPage.js [hunk 4]
```diff
@@ -48,7 +50,18 @@ const components = [
...
+        src={image1}
+        srcSet={`${image1}?resize=320px:320px&quality=100 320w,
+          ${image1}?resize=520px:520px&quality=100 520w,
+          ${image1}?resize=1024px:1024px&quality=100 1024w,
+          ${image1}?resize=1920px:1920px&quality=100 1920w`}
+        // If the viewport is wider than 1140px, then this image will
+        // take up ≈765px of the viewport. If the viewport
+        // is wider than 805px it will take up ≈ all but 292px
+        // Otherwise, it will take up ≈ all but 72px.
+        sizes={`(min-width: 1140px) 765px,
```
Comment (matthewdavidson) [inline packages/bpk-docs/src/pages/ImagesPage/ImagesPage.js]: use tokens for these breakpoint values

**Pull request merged**